src/main/scripts/addon

```diff
@@ -125,4 +126,4 @@ JAVA_OPTS="$JAVA_OPTS --add-opens=java.base/sun.net.www.protocol.jar=ALL-UNNAMED
 JAVA_OPTS="$JAVA_OPTS --add-opens=java.base/sun.nio.ch=ALL-UNNAMED"
 JAVA_OPTS="$JAVA_OPTS --add-opens=java.base/sun.security.util=ALL-UNNAMED"
 
-eval exec \"$_RUNJAVA\" "$JAVA_OPTS" -jar \"$PLF_HOME/addons/addons-manager.jar\" "$@"
\ No newline at end of file
+eval exec \"$_RUNJAVA\" "$JAVA_OPTS" "$EXO_AM_OPTS" -jar \"$PLF_HOME/addons/addons-manager.jar\" "$@"
```

Comment:
This change is useless
```suggestion
eval exec \"$_RUNJAVA\" "$JAVA_OPTS" -jar \"$PLF_HOME/addons/addons-manager.jar\" "$@"
```

Reply:
Hi @fdrouet, I checked with the last suggestion and it worked. Thanks